integration/sql/impl/src/lineage.rs

```diff
@@ -79,6 +79,8 @@ pub struct DbTableMeta {
     pub database: Option<String>,
     pub schema: Option<String>,
     pub name: String,
+    pub has_namespace: bool,
+    pub has_field_schema: bool,
```

Comment:
We need to at least document what true and false values mean here, or change the names so that's obvious. 

By looking at it I don't know if extractor should query for table schema if value `has_field_schema` is true or false. The same for `has_namespace`.